main.go

```diff
@@ -136,6 +136,9 @@ func run() {
 		}()
 	}
 
+	// 等待网络连接
+	util.WaitInternet(dns.Addresses, util.GetDefaultDNS(conf.Lang, *customDNS))
```

Comment:
感觉一个检查网络的函数还得提前把备用 DNS 传进去很奇怪；维护一个 DNS 列表，如果传了自定义 DNS 把自定义加进去，否则根据语言把默认 DNS 加进去。这样应该好些。

```suggestion
	util.AddDNS(conf.Lang, *customDNS)

	// 等待网络连接
	util.WaitInternet(dns.Addresses)
```